tests/support/server.tcl

```diff
@@ -418,7 +418,7 @@ proc start_server {options {code undefined}} {
                 set baseconfig $value
             }
             "overrides" {
-                set overrides $value
+                set overrides [concat $overrides $value]
```

Comment:
`overrides` is already a list, do we have a use case that passes two override lists?
i don't see one..

Reply:
Maybe there is a better way, but right now the cluster server setup add a set of overrides but the test can add another set.